cmd/edit/cluster/cmd.go

```diff
@@ -654,6 +673,26 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	// AutoNode configuration
+	autonodeMode, autonodeRoleArn, err := setAutoNode(r, cmd, cluster, args.autonode, args.autonodeRoleARN)
+	if err != nil {
+		r.Reporter.Errorf("%s", err)
+		os.Exit(1)
+	}
+	if interactive.Enabled() && aws.IsHostedCP(cluster) {
+		interactiveMode, interactiveRoleArn, err := autonodeInteractivePrompt(r, cmd, cluster)
```

Comment:
Could we change the names of these to just `autonodeMode` and `autonodeRoleArn` like the above ones? `interactiveMode` makes me immediately think of `-i` or `--interactive` mode rather than the auto node gathered from interactive

Reply:
fixed on last commit